gbpservice/nfp/config/mode_shift.sh

```diff
@@ -0,0 +1,164 @@
+#! /bin/bash
```

Comment:
This script should go into scripts directory, not config. Create a new scripts directory under gbpservice/nfp.